monai/bundle/scripts.py

```diff
@@ -192,6 +193,13 @@ def _download_from_ngc(
     extractall(filepath=filepath, output_dir=extract_path, has_base=True)
 
 
+def _download_from_huggingface_hub(repo: str, download_path: str, filename: str, version: str) -> None:
```

Comment:
```suggestion
def _download_from_huggingface_hub(repo_id: str, download_path: str, filename: str, version: str) -> None:
```

for consistency with the rest, I would rename it  `repo_id`